python/simpler/worker.py

```
@@ -8171,6 +8211,179 @@ def _submit_locked(self, callable, args, config) -> RunHandle:
+                        self._l2_launch_front_locked(successor)
+                continue
+
+            if not block:
+                return False
+            if deadline is not None:
+                remaining = deadline - time.monotonic()
+                if remaining <= 0:
+                    return False
+                time.sleep(min(remaining, _RUN_HANDLE_WAIT_RECHECK_S))
+
+    def _submit_l2_locked(self, callable, args, cfg: CallConfig) -> RunHandle:
```

Ruff (0.16.0)

[error] Function argument `callable` is shadowing a Python builtin

(A002)